src/View/Widget/MultiCheckboxWidget.php

```diff
@@ -112,7 +112,8 @@ public function render(array $data, ContextInterface $context)
             'disabled' => null,
             'val' => null,
             'idPrefix' => null,
-            'templateVars' => []
+            'templateVars' => [],
+            'label' => true
```

Comment:
The value of `label` would be the same as value of `labelOptions` passed to FormHelper::input() so better to keep the name same.